TreeGrid.php

```diff
@@ -244,6 +248,9 @@ public function renderTableRow($model, $key, $index)
 
         $parentId = ArrayHelper::getValue($model, $this->parentColumnName);
         if ($parentId) {
+            if('collapsed' == ArrayHelper::getValue($this->pluginOptions, 'initialState')){
```

Comment:
I know, but i think that yii2 does not use this pattner

https://github.com/yiisoft/yii2/blob/master/framework/base/Model.php#L778